support-frontend/app/controllers/StripeController.scala

```diff
@@ -52,7 +52,7 @@ class StripeController(
 
     val v2RecaptchaToken = request.body.token
 
-    val cloudwatchEvent = createSetupIntentRequest(stage, "v2Recaptcha")
+    val cloudwatchEvent = AwsCloudWatchMetricSetup.createSetupIntentRequest(stage, "v2Recaptcha")
     AwsCloudWatchMetricPut(client)(cloudwatchEvent)
```

Comment:
the domain specific convenience methods are all moved to `AwsCloudWatchMetricSetup` so that the actual `Put`ter can be generic